src/abstractions/Backend.Fx/Patterns/EventAggregation/Integration/InMemoryMessageBus.cs

```diff
@@ -6,14 +6,27 @@
 
     public class InMemoryMessageBus : MessageBus
     {
+        private readonly InMemoryMessageBusChannel _channel;
+
+        public InMemoryMessageBus()
+        {
+            _channel = new InMemoryMessageBusChannel();
+        }
+        
+        public InMemoryMessageBus(InMemoryMessageBusChannel channel)
+        {
+            _channel = channel;
+        }
+        
         public override void Connect()
         {
+            _channel.MessageReceived += ChannelOnMessageReceived;
```

Comment:
due to the singleton nature of this class it won't be a huge problem, but since MessageBus is also IDisposable we should unsubscribe from the event there to make it perfect

Reply:
Thanks. I added the dispose.